app/js/model/list_model.js

```diff
@@ -2,8 +2,34 @@ import { Model } from 'components/fxos-mvc/dist/mvc';
 import { AppsHelper, IconHelper } from 'js/lib/helpers';
 
 export default class ListModel extends Model {
+  getAllApps() {
+    return new Promise((resolve, reject) => {
+      AppsHelper.getAllApps().then(allApps => {
+        resolve(allApps);
+      });
+    });
+  }
+
+  filterCustomizerAddon(apps) {
+    return apps.filter(app =>
+      app.manifest.role == 'addon' &&
+      app.manifest.name == 'Customizer' &&
+      app.manifestURL == 'app://customizer.gaiamobile.org/manifest.webapp');
+  }
+
+  getCustomizerAddOn(allApps) {
+    return new Promise((resolve, reject) => {
+      var addOnList = this.filterCustomizerAddon(allApps);
+      if (addOnList && addOnList.length == 1) {
+        resolve(addOnList[0]);
+      } else {
+        reject(new Error('Cannot fetch customizer add on'));
+      }
+    });
+  }
+
   filterApps(apps) {
-    var excludedApps = ['Built-in Keyboard', 'Settings'];
+    var excludedApps = ['Built-in Keyboard', 'Settings', 'Customizer Launcher'];
```

Comment:
Also, let's filter excluded apps by manifestURL instead of name. This is what we do in Customizer also:

https://github.com/fxos/customizer/blob/master/app/js/app.js#L4